opencti-platform/opencti-front/src/private/components/nav/TopBar.tsx

```diff
@@ -496,8 +497,17 @@ const TopBarComponent: FunctionComponent<TopBarProps> = ({
                 {t('Profile')}
               </MenuItem>
               <MenuItem onClick={handleOpenDrawer}>{t('Feedback')}</MenuItem>
-              <MenuItem id="logout-button" onClick={() => handleLogout()}>
-                {t('Logout')}
+              <MenuItem id="logout-button">
+                <MuiLink
+                  underline="none"
+                  color="secondary"
+                  href="/logout"
+                  // user will be redirected to login, but we do not want to keep the referrer url
```

Comment:
```suggestion
                  // user will be redirected to logout, but we do not want to keep the referrer url
```
No ?

Reply:
hm I meant he's redirected to /logout ... which shows him the login screen.

Reply:
Ahhhh! Suggestion updated